scripts/timeit.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import re
```

Comment:
flake8 is not happy:
scripts/timeit.py:3:1: I201 Missing newline before sections or imports.
scripts/timeit.py:81:80: E501 line too long (85 > 79 characters)

Reply:
done, thanks